packages/nextlove/src/with-route-spec/index.ts

```diff
@@ -49,6 +49,7 @@ export const createWithRouteSpec: CreateWithRouteSpecFunction = ((
     authMiddlewareMap = {},
     globalMiddlewares = [],
     shouldValidateResponses,
+    shouldValidateGetRequestBody = true,
```

Comment:
I believe the default should be `false`

Reply:
The current behavior is to validate get request bodies right? So this default is chosen to keep the current behavior with the intention to make it false with a major version bump. (Please confirm I have that right.)

I'll create an issue for the next major version to collect breaking changes to include in the next major.